pkg/action/upgrade_test.go

```diff
@@ -623,3 +660,37 @@ var _ = Describe("Runtime Actions", func() {
 		})
 	})
 })
+
+func createDummyFile(fs vfs.FS, sizeMb int64) string {
```

Comment:
`os.Truncate` doesn't take a filesystem as an argument. So, I would have to create the file somewhere in tmp, then read it in memory and then write it as a file in the test fs.
It will save the look to generate the data but will introduce more code to move the file in the test fs.
I can change it if you think it's better, I'm not sure myself.

Reply:
Should be like this I guess

```
f, err := fs.Create("/folder/file")
Expect(err).ShouldNot(HaveOccurred())
err = f.Truncate(1024)
Expect(err).ShouldNot(HaveOccurred())
f.Close()
```

Reply:
Oh, file object also has a `Truncate` method. Nice! Fixing.